go/ngql/types.go

```diff
@@ -203,10 +203,17 @@ func structToGQLObject(nomsType *types.Type, tm *typeMap) *graphql.Object {
 			structDesc.IterFields(func(name string, nomsFieldType *types.Type) {
 				fieldType := nomsTypeToGraphQLType(nomsFieldType, false, tm)
 
+				if nonNull, ok := fieldType.(*graphql.NonNull); ok {
```

Comment:
Wait - did you deliberately not update the schema generation code? That still produces not-nullable types.

I mean, this seems fine (until we have optional types), just pointing it out.